ingestion-beam/src/main/java/com/mozilla/telemetry/util/Json.java

```diff
@@ -56,7 +56,7 @@ public static JSONObject readJSONObject(byte[] data) throws IOException {
     // Read data into a tree
     TreeNode tree = MAPPER.readTree(data);
     // Check that we have an object, because treeToValue won't
-    if (!tree.isObject()) {
+    if (tree == null || !tree.isObject()) {
```

Comment:
`tree.isObject` was throwing a NullPointerException in the case of an empty byte array.